app/controllers/api/base_controller.rb

```diff
@@ -121,7 +121,7 @@ def service_unavailable(exception = nil)
     end
 
     def process_resource_error(options = { })
-      resource = options[:resource] || get_resource
+      resource = options[:resource] || get_resource(options)
```

Comment:
Given what I see on https://github.com/Katello/katello/pull/6543/files, isn't `options[:message]` what you want to pass here? Otherwise you're just passing a hash and calling `raise()` on a hash instead of a string